### DISCUSSION TITLE
MRTK openXR on MetaPro and Focus3

### DISCUSSION THREAD

**Discussion by sebb81** [microsoft/MixedRealityToolkit-Unity#11637] [Q&A]
Hello, I'm using MRTK2 + Unity on a Meta Pro headset with pass-through enabled, and it's working great! (using the Oculus XR plugin and Oculus Integration).

Now, I would like to switch to using OpenXR exclusively so that I can install it on Meta and HTC Focus 3 headsets. I downloaded the OpenXR support via the Mixed Reality Toolkit and modified the XR Plugin Management to use OpenXR. However, when I try to build, I receive the following error message:

**_Library\PackageCache\com.microsoft.mixedreality.toolkit.foundation@f2323a9b7aec\Providers\OpenXR\Scripts\OpenXRControllerModelProvider.cs(8,30): error CS0234: The type or namespace name 'OpenXR' does not exist in the namespace 'Microsoft.MixedReality' (are you missing an assembly reference?)_**

Have you ever tried this setup? Is it possible? 
Perhaps I'm doing something wrong as I'm relatively new to this field. 

Thanks in advance.
Seb

**Comment by VelazcoJD** [comment]
I am getting the same issue trying to build the app for the Magic Leap. Did you ever figure out the issue?
